src/main/java/org/swisspush/redisques/RedisQues.java

```diff
@@ -1302,4 +1373,60 @@ private QueueConfiguration findQueueConfiguration(String queueName) {
         }
         return null;
     }
+
+    /**
+     * Retrieve the number of items of the requested queue
+     */
+    private void getQueueItemsCount(Message<JsonObject> event) {
+        String queue = event.body().getJsonObject(PAYLOAD).getString(QUEUENAME);
+        redisAPI.llen(queuesPrefix + queue, new GetQueueItemsCountHandler(event));
+    }
+
+    /**
+     * Retrieve the number of items of many queues
+     */
+    private void getQueuesItemsCount(Message<JsonObject> event) {
+        Result<Optional<Pattern>, String> filterPattern = MessageUtil.extractFilterPattern(event);
+        getQueuesItemsCount(event, filterPattern);
+    }
+
+    /**
+     * Retrieve the size of the queues matching the given filter pattern
+     */
+    private void getQueuesItemsCount(Message<JsonObject> event, Result<Optional<Pattern>, String> filterPatternResult) {
+        if (filterPatternResult.isErr()) {
+            event.reply(createErrorReply().put(ERROR_TYPE, BAD_INPUT)
+                .put(MESSAGE, filterPatternResult.getErr()));
+        } else {
+            redisAPI.zrangebyscore(List.of(queuesKey, String.valueOf(getMaxAgeTimestamp()), "+inf"),
+                new GetQueuesItemsCountHandler(event, filterPatternResult.getOk(), luaScriptManager,
+                    queuesPrefix));
+        }
+    }
+
+    /**
+     * Retrieve the queue statistics info of the requested queues
+     * @param event
+     */
+    private void getQueuesStatistics(Message<JsonObject> event) {
+        Result<Optional<Pattern>, String> filterPattern = MessageUtil.extractFilterPattern(event);
+        getQueuesStatistics(event, filterPattern);
+    }
+
+    /**
+     * Retrieve the queue statistics info of the requested queues filtered by the
+     * given filter pattern.
+     */
+    private void getQueuesStatistics(Message<JsonObject> event,
+        Result<Optional<Pattern>, String> filterPattern) {
+        if (filterPattern.isErr()) {
```

Comment:
Same as [comment 803620543](https://github.com/swisspush/vertx-redisques/pull/85/files#r803620543).

Reply:
As well here. As already written. Let us aply the GoogleCodeFormatter once and for all and stop taste of formatting discussions.